index.html

```diff
@@ -73,6 +74,57 @@ <h2 class="about-l-header">Skills</h2>
               <p class="work-text"><img src="multimedia/down-arrow.png" alt="Gray down arrow">Work</p>
           </div>        
         </section>
-      </div>  
+        <section class="recent-works">
+          <h2 class="recent-works-header">MY RECENT WORK</h2>
+          <hr> 
+          <section class="work-card">
```

Comment:
## Optional.
We should never use `<section>` inside another `<section>`. Since each project is a stand-alone item inside the main section, you can consider using `<article>` tag.

Reply:
@muneebulrehman Done, thank you for this recommendation :+1: